src/main/java/org/opentripplanner/gtfs/mapping/TransferMapper.java

```diff
@@ -64,6 +64,17 @@
    */
   private static final int FORBIDDEN = 3;
 
+  /**
+   * Passengers can transfer from one trip to another by staying onboard the same vehicle.
+   */
+  private static final int STAY_SEATED = 4;
```

Comment:
Since this isn't in the mainline GTFS spec would you be able to add a link to where it is defined?

Reply:
Added